integration_tests/e2e/signIn.cy.ts

```diff
@@ -41,6 +41,7 @@ context('SignIn', () => {
 
   it('User can manage their details', () => {
     cy.signIn()
+    cy.task('stubAuthManageDetails')
```

Comment:
this was the only place that manage details was needed, so made sense to just call it here instead